charts/dbildungs-iam-keycloak/dev-realm-spsh.json

```diff
@@ -799,6 +818,109 @@
           "microprofile-jwt"
         ]
       },
+      {
+        "id": "1dcfa102-dad9-4cf4-b0ff-dcc945e4a2ca",
+        "clientId": "spsh-service",
+        "name": "SPSH Service Client",
+        "description": "",
+        "rootUrl": "",
+        "adminUrl": "",
+        "baseUrl": "",
+        "surrogateAuthRequired": false,
+        "enabled": true,
+        "alwaysDisplayInConsole": false,
+        "clientAuthenticatorType": "client-jwt",
+        "secret": "**********",
```

Comment:
@tikle-cap Please replace asterisks with some plain text client secret (e. g. "spsh-service-client-secret") once we are sure this cannot be abused.

Reply:
replaced